src/v2/view-builder/components/AuthenticatorVerifyOptions.js

```diff
@@ -19,7 +19,12 @@ const AuthenticatorRow = View.extend({
           </div>\
         </div>\
         <div class="authenticator-description">\
-          <h3 class="authenticator-label">{{label}}</h3>\
+          <div class="{{#if sublabel}}authenticator-description--with-sublabel{{/if}}">\
```

Comment:
The screenshots are different - So the mark-up needs to be - 
This is for Enrollment -
![image](https://user-images.githubusercontent.com/62976351/88111462-cc1e6900-cb62-11ea-8001-11dce98c2a7d.png)

Reply:
oh, correct.. I was thinking using similar dom structure of

```
          {{#if description}}\
            <p class="authenticator-description--text">{{description}}</p>\
          {{/if}}\
```

Reply:
The mark-up is very similar. There's a `div` that wraps `h3` and `p` in verify options that accommodates stacking on the left side. The only other difference is that I didn't want to call it description because, it will display labels if not may cause confusion with enroll. Hence opted to call it `sublabel` instead.